style.css

```diff
@@ -16,8 +16,10 @@ h1 {
 #target {
   border: 2px dashed #ccc;
   color: #ccc;
+  cursor: pointer;
   font-size: 1.5em;
-  line-height: 4;
+  line-height: 1.5em;
+  padding: 1.5em;
```

Comment:
I tweaked the line-height and padding so that this wouldn't look bad on mobile if the text wraps.